src/components/tooltip/Marker.js

```diff
@@ -208,496 +62,6 @@ function _maybeFormatDuration(
-    return (
-      <div className="tooltipDetailsBackTrace" key="backtrace">
-        {data.type === 'Styles' || marker.name === 'Reflow' ? (
-          <h2 className="tooltipBackTraceTitle">
-            First invalidated {formatNumber(causeAge)}ms before the flush, at:
-          </h2>
-        ) : null}
-        <Backtrace
-          maxHeight="30em"
-          stackIndex={cause.stack}
-          thread={thread}
-          implementationFilter={implementationFilter}
-        />
-      </div>
-    );
-  }
-  return null;
-};
-
-function getMarkerDetails(
-  marker: Marker,
-  thread: Thread,
-  implementationFilter: ImplementationFilter,
-  zeroAt: Milliseconds
-): React.Node {
-  const data = marker.data;
-  let tooltipDetails;
-
-  if (data) {
-    switch (data.type) {
-      case 'FileIO': {
-        tooltipDetails = (
-          <TooltipDetails>
-            <TooltipDetail label="Operation">{data.operation}</TooltipDetail>
-            <TooltipDetail label="Source">{data.source}</TooltipDetail>
-            <TooltipDetail label="Filename">{data.filename}</TooltipDetail>
-          </TooltipDetails>
-        );
-        break;
-      }
-      case 'UserTiming': {
-        tooltipDetails = (
-          <TooltipDetails>
-            <TooltipDetail label="Name">{data.name}</TooltipDetail>
-          </TooltipDetails>
-        );
-        break;
-      }
-      case 'Text': {
-        tooltipDetails = (
-          <TooltipDetails>
-            <TooltipDetail label="Name">{data.name}</TooltipDetail>
-          </TooltipDetails>
-        );
-        break;
-      }
-      case 'Log': {
-        tooltipDetails = (
-          <TooltipDetails>
-            <TooltipDetail label="Module">{data.module}</TooltipDetail>
-            <TooltipDetail label="Name">{data.name}</TooltipDetail>
-          </TooltipDetails>
-        );
-        break;
-      }
-      case 'GCMinor': {
-        if (data.nursery) {
-          const nursery = data.nursery;
-          switch (nursery.status) {
-            case 'complete': {
-              // Don't bother adding up the eviction time without the
-              // CollectToFP phase since that's the main phase.  If it's
-              // missing then there's something wrong with the profile and
-              // we'd only get bogus data.  All these times are in
-              // Milliseconds
-              const evictTimeMS = nursery.phase_times.CollectToFP
-                ? _sumMaybeEntries(nursery.phase_times, [
-                    'TraceValues',
-                    'TraceCells',
-                    'TraceSlots',
-                    'TraceWholeCells',
-                    'TraceGenericEntries',
-                    'MarkRuntime',
-                    'MarkDebugger',
-                    'CollectToFP',
-                  ])
-                : undefined;
-              tooltipDetails = (
-                <TooltipDetails>
-                  <TooltipDetail label="Reason">{nursery.reason}</TooltipDetail>
```

Comment:
This `nursery.reason` field is not present in the new code inside `getGCMinorDetails` function. Is there a reason not to print it anymore?